bundles/org.openhab.binding.myuplink/src/main/java/org/openhab/binding/myuplink/internal/MyUplinkBindingConstants.java

```diff
@@ -0,0 +1,179 @@
+/**
+ * Copyright (c) 2010-2024 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.myuplink.internal;
+
+import java.time.Instant;
+import java.util.Set;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.openhab.core.thing.ThingTypeUID;
+
+/**
+ * The {@link MyUplinkBindingConstants} class defines common constants, which are
+ * used across the whole binding.
+ *
+ * @author Alexander Friese - Initial contribution
+ */
+@NonNullByDefault
+public class MyUplinkBindingConstants {
+
+    public static final String BINDING_ID = "myuplink";
+
+    // List of main device types
+    public static final String DEVICE_ACCOUNT = "account";
+    public static final String DEVICE_GENERIC_DEVICE = "genericDevice";
```

Comment:
See earlier comment about naming convention (thid id as to be adapted on more places)

Reply:
fixed